cmd/cosign/cli/sign/sign_blob.go

```diff
@@ -117,26 +124,83 @@ func SignBlobCmd(ro *options.RootOptions, ko options.KeyOpts, payloadPath string
 		}
 		ui.Infof(ctx, "tlog entry created with index: %d", *entry.LogIndex)
 		signedPayload.Bundle = cbundle.EntryToBundle(entry)
+		rekorEntry = entry
 	}
 
 	// if bundle is specified, just do that and ignore the rest
 	if ko.BundlePath != "" {
-		signedPayload.Base64Signature = base64.StdEncoding.EncodeToString(sig)
+		if ko.UsePBBundleFormat {
+			vm := pbbundle.VerificationMaterial{}
 
-		certBytes, err := extractCertificate(ctx, sv)
-		if err != nil {
-			return nil, err
-		}
-		signedPayload.Cert = base64.StdEncoding.EncodeToString(certBytes)
+			certBytes, err := sv.Bytes(ctx)
+			if err != nil {
+				return nil, fmt.Errorf("error getting signer: %w", err)
+			}
+			certChain, err := protobundle.GenerateX509CertificateChain(certBytes)
```

Comment:
A recent decision was to have this only be the non-trusted certificates, and any CA certs that appear in the root of trust should not be included - https://github.com/sigstore/protobuf-specs/blob/main/protos/sigstore_common.proto#L171-L188

We can either:
* Load in just the leaf certificate and add a TODO to handle private deployments
* Just cut out the root and include leaves and intermediates
* Cut out any CA certs that are returned from these two funcs - https://github.com/sigstore/cosign/blob/main/internal/pkg/cosign/fulcio/fulcioroots/fulcioroots.go#L37-L57